app/views/layouts/application.html.haml

```diff
@@ -28,5 +28,5 @@
     = render 'shared/environment_variables'
     = render 'shared/messages'
     = javascript_include_tag "https://maps.googleapis.com/maps/api/js?key=#{ENV['GOOGLE_MAPS_API_KEY']}"
-    = javascript_pack_tag 'application'
+    -# = javascript_pack_tag 'application'
```

Comment:
Is there any necessary JS in this? or what it just old compass things?

Reply:
I'm 95% sure it's old stuff we don't use anymore. Commenting it like this doesn't break anything from what I can tell. But that is why I commented rather than removed, easy to put it back in later.

Reply:
It looks like its turbo links and uppy config. I thought uppy was still in use in the admin side. Was this removed due to JS errors?

Reply:
nice investigation, sounds like you sorted it out, i was missing the knowledge of the hotjar changes around those popup links.